lib/OpenQA/Utils.pm

```diff
@@ -1088,4 +1089,13 @@ sub change_sec_to_word {
     return $time_word;
 }
 
+sub handle_plus_in_settings {
```

Comment:
It's better than nothing though I expected to extract a bigger part of common functionality ;) Ok for this PR but I hope you can look into an improvement as a follow-up

Reply:
See my other comment, this belongs in `Script`.

The uppercase key handling would be another straightforward candidate for deduplication. Although I could see that you might want to do this in separate PRs.

Reply:
I thought `Script` was for additional scripts like the one to clone jobs. But this function is used in the overall result set (which is used in the controller).

Reply:
I could move this function to ExpandPlaceholder.pm,  and rename that file (JobSettings.pm?). Then the functions in that file are all used to handle the job settings. Is it ok? @Martchus @kalikiana 
@okurz I got you meaning. I will look into that but I am not sure if I could do that perfectly, I tried to do that before, but failed.  :(

Reply:
I understand. Your proposed solution sounds good to me.